contrib/endpoints/src/api_manager/proto/server_config.proto

```diff
@@ -143,6 +146,12 @@ message ApiAuthenticationConfig {
   bool force_disable = 1;
 }
 
+// Server config for API Authorization via Firebase Rules
+message ApiCheckSecurityRulesConfig {
+  // Allows to disable the API authorization
+  bool force_disable = 1;
+}
```

Comment:
If that is the case,   it will be easier for you to integrate with ESP if you default to be disabled,  only enable it from server_config.